src/librustc/middle/liveness.rs

```diff
@@ -1265,22 +1259,23 @@ impl<'a, 'tcx> Liveness<'a, 'tcx> {
         }
     }
 
-    fn access_var(&mut self, id: NodeId, nid: NodeId, succ: LiveNode, acc: u32, span: Span)
+    fn access_var(&mut self, hir_id: HirId, nid: NodeId, succ: LiveNode, acc: u32, span: Span)
```

Comment:
You could change `nid` to `HirId` too, I think.

Reply:
This doesn't feel like an organic part of this somewhat-narrowly-focused cleanup PR to me (value called as `nid` is ultimately coming out of a `Def::Local(NodeId)` variant, in contrast to `hir::Pat` already having a `hir_id` field)?